src/api/roles/shared/client_coach_relationship.py

```
@@ -0,0 +1,56 @@
+from src.database.account.models import Account
+from src.database.client.models import Client
+from src.database.coach.models import Coach
+from src.database.coach_client_relationship.models import ClientCoachRequest, ClientCoachRelationship
+
+
+from src.api.roles.shared.domain import DeleteRequestResponse
+router = APIRouter(prefix="roles/shared/client_coach_relationship", tags=["shared", "client_coach_relationship"])
+
+
+@router.delete("/delete_coach_request/{request_id}", response_model=DeleteRequestResponse)
+def delete_coach_request(request_id: int, db = Depends(get_session), acc: Account = Depends(get_client_account)):
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)